libs/designer-ui/src/lib/mixedinputeditor.tsx

```diff
@@ -0,0 +1,110 @@
+import type React from 'react';
+import { useEffect, useState } from 'react';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+import type { ChangeHandler, ValueSegment } from '@microsoft/logic-apps-shared';
+import { SchemaEditor } from './schemaeditor';
+import { FloatingActionMenuInputs } from './floatingactionmenu/floatingactionmenuinputs';
+
+interface MixedInputEditorProps {
+  initialValue: ValueSegment[];
+  onChange: ChangeHandler;
+  supportedTypes?: string[];
+  isRequestApiConnectionTrigger?: boolean;
+  label?: string;
+  readonly?: boolean;
+  useStaticInputs: boolean;
+}
+
+// Function to check if `initialValue` contains "useSchemaEditor": true
+const checkIsSchemaEditor = (initialValue: ValueSegment[]): boolean => {
+  return initialValue.some((segment) => {
+    try {
+      const parsedValue = JSON.parse(segment.value);
+      return parsedValue?.useSchemaEditor === true;
+    } catch (error) {
+      console.error('Error parsing value segment:', error);
```

Comment:
I generally avoid console logs like this. If it's a meaningful log that we'd like visibility on it I would use the logger service for the project. If it's for debug purposes only I would remove it